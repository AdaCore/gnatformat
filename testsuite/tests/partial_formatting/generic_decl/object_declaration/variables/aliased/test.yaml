description: ObjDecl formatting from line selection.
driver: partial_gnatformat
args:
  [
    "test.ads",
    "-SL 74",
    "-EL 75",
  ]
status_code: 1