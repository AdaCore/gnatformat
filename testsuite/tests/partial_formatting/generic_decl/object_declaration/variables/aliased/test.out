<CompilationUnit test.ads:26:1-87:25>
test.ads(<ObjectDecl ["Hello"] test.ads:74:4-74:49>) - 74:1-74:49
^
   Hello : aliased String := "Hi, world.";$

